lib/output.py

```diff
@@ -107,6 +107,12 @@ def write_embroidery_file(file_path, stitch_plan, svg, settings={}):
 
     try:
         pyembroidery.write(pattern, file_path, settings)
+    except AssertionError:
```

Comment:
I'd love this to be translatable. What if we check for some words here?

Reply:
How about if PyEmbroidery throws a TooManyColorChangesError rather than an AssertionError?  Then we can still catch it in Ink/Stitch and give a translated error message.  We'd have to define TooManyColorChangesError in PyEmbroidery, of course.

This isn't even what `assert` is for, it's only supposed to be used for things that should never happen, like a bug in the code itself.  I know this will make us diverge from upstream pyembroidery a bit, but I think it's worth it to do this right.